src/preact/utils/docInfo.js

```diff
@@ -0,0 +1,46 @@
+import {urlUtils} from './url';
+
+export const docInfo = {
+  /**
+   * @param {string} metaName
+   * @return {string|null}
+   */
+  getMetaByName(metaName) {
+    const metas = self.document.head.querySelectorAll('meta[name]');
+    return (
+      Array.from(metas.values())
+        .find(
```

Comment:
Nit: this is my preference, but I hate using array methods when something is easily expressed as a for-loop.

Reply:
https://github.com/ampproject/amphtml/pull/37127#discussion_r791893151
I would like to reduce the number of "preferences" blocking this PR.

Reply:
@jridgewell this is on me, I wrote/suggested this code in place of an existing for-loop. Opposite preferences 😬

Reply:
Nits are never blocking.